examples/go.mod

```diff
@@ -11,22 +11,31 @@ require (
 	github.com/lib/pq v1.10.2
 	github.com/mattn/go-sqlite3 v1.14.9
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.2.1
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
+	gorm.io/driver/sqlite v1.3.1
+	gorm.io/gorm v1.23.2
+)
+
+// Require platform-lib dependencies. Note that the versions don't matter here
+// since all get replaced by the `replace` directive below. The versions here
+// should match some valid version to make the IDE happy, but they're ignored
+// by Go when compiling.
```

Comment:
Because of the `replace` directives below, none of the changes here really make a difference. The main thing is that I added some comments to clarify what's happening.